.github/workflows/bump_license_year.yml

```diff
@@ -6,6 +6,7 @@
 #############################################################################
 name: "Bump license year"
 on:
+  workflow_dispatch: null
```

Comment:
Please specify branch here, like:
```
workflow_dispatch:
  branches: [ master ]

```
If cron only runs on master, so should dispatched job.

Reply:
done

Reply:
Branches option for workflow_dispatch sees to deprecated, actionlint failed on this. I restored `null`.